<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicView</class>
 <widget class="QDockWidget" name="PicView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>817</height>
   </rect>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Pic'eau Plugin</string>
  </property>
  <widget class="QWidget" name="plot_window">
   <widget class="QWebView" name="plot_WV">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stat_descr">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>730</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Statistiques Descriptives</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_trace">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Voir la chronique</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetBSS">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
